### DISCUSSION TITLE
Using header.php file for the site Header instead of using the default theme builder

### DISCUSSION THREAD

**Discussion by r-interactive** [elementor/elementor#24788] [Developers Community]
### Start a discussion

Hi,

Is there a way to use my header.php file (with a custom menu using wp_nav_menu(), php, html, css, js) for the site header instead of using the elementor theme builder?
If so, how can we integrate that header.php file and have elementor to use that instead of the default theme builder?
Can someone point me to the right direction of connecting the php file and elementor? 

thank you!

**Comment by perks98** [comment]
If you are using your own header.php file make sure you are using a child theme, then add your header.php to the child theme and it should use that file rather than the parent theme header.php file.

**Reply by r-interactive** [reply]
Thank you @perks98
